### PR TITLE
API Generator: Add missing `scope` argument and filter to `<entity>BySlug` query

### PR DESCRIPTION
**Pull request by kaufmo**
- [x] Add changeset (if necessary)

### CODE REVIEW

File: .changeset/three-eagles-cheat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": patch
+---
+
+api-generator: add scope argument and filter for slug query
```
Comment (johnnyomair) [inline .changeset/three-eagles-cheat.md:5]:
```suggestion
API Generator: Support `scope` argument and filter in `<entity>BySlug` query
```

File: .changeset/three-eagles-cheat.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": patch
```
Comment (johnnyomair) [inline .changeset/three-eagles-cheat.md:2]:
```suggestion
"@comet/cms-api": minor
```

IMO this is a feature, what do you think?

Comment (kaufmo) [reply .changeset/three-eagles-cheat.md:2]: as I talked with @nsams he thougt this is a bug, minor is ok for me?

Comment (nsams) [reply .changeset/three-eagles-cheat.md:2]: Whatever

**Review by nsams — APPROVED** [review]

File: .changeset/three-eagles-cheat.md (hunk 2 above)
Comment (kaufmo) [reply .changeset/three-eagles-cheat.md:2]: @johnnyomair what do you want? minor or patch?

Comment (johnnyomair) [reply .changeset/three-eagles-cheat.md:2]: Patch is fine for me. Will change the changeset accordingly.

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**